android/modules/filesystem/src/java/ti/modules/titanium/filesystem/ResourceHelper.java

```diff
@@ -0,0 +1,28 @@
+package ti.modules.titanium.filesystem;
```

Comment:
That will create a circular dependency between 'titanium' and 'titanium-filesystem' modules.